terraform/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "jenkins_public_ip" {
```

Comment:
@Vermyndax is this the 1:1 tag used by terraform-inventory and in turn ansible?

Reply:
Yes.